docs/config.md

```diff
@@ -221,5 +221,12 @@ improvement, as well as a reduction in memory usage.
 - `DisableNatPortMap`
 Disable NAT discovery.
 
+- `DisableRelay`
+Disables the p2p-circuit relay transport.
```

Comment:
Will this disable relays entirely or just disable listening on relayed addresses? There are three cases:

1. Listen using relays.
2. Act as a relay.
3. Connect to nodes through relays.

It would be nice to be able to enable/disable these independently. Can we currently do that?